# wheels control loop PID
front_right_wheel_joint: {p: 1.0, i: 1.0, d: 0.0, i_max: 100.0, i_min: -100.0}
front_left_wheel_joint: {p: 1.0, i: 0.0, d: 0.0, i_max: 0.0, i_min: 0.0}
rear_right_wheel_joint: {p: 1.0, i: 0.0, d: 0.0, i_max: 0.0, i_min: 0.0}
rear_left_wheel_joint: {p: 1.0, i: 0.0, d: 0.0, i_max: 0.0, i_min: 0.0}

lpf: {front_right: 0.2, front_left: 0.2, rear_right: 0.2, rear_left: 0.2}

reverse_command: {front_right: false, front_left: false, rear_right: false, rear_left: false}
reverse_feedback: {front_right: true, front_left: true, rear_right: true, rear_left: true}

# delta t for velocity deriative calculation (seconds)
velocity_delta_t: 0.1

# add bias to servo command. lets say motor only start
# to react to commands > 1600 or < 1400. Than a bias
# of 100 needs to be added to each command. Enter 0 to disable
ric_servo_bias: 100

motor_max_rpm: 150